dropwizard-service-discovery-bundle/src/main/java/io/appform/dropwizard/discovery/bundle/id/IdGenerator.java

```diff
@@ -273,11 +248,33 @@ private static IdValidationState validateId(List<IdValidationConstraint> inConst
                 .filter(constraint -> !constraint.isValid(id))
                 .findFirst()
                 .orElse(null);
-        if(null != failedLocalConstraint) {
+        if (null != failedLocalConstraint) {
             return failedLocalConstraint.failFast()
                     ? IdValidationState.INVALID_NON_RETRYABLE
                     : IdValidationState.INVALID_RETRYABLE;
         }
         return IdValidationState.VALID;
     }
+
+    private enum IdValidationState {
+        VALID,
+        INVALID_RETRYABLE,
+        INVALID_NON_RETRYABLE
+    }
+
+    private static final class IdInfo {
```

Comment:
`@Value`

Reply:
This is just reformatting of code. @Value is not needed here

Reply:
Done